File: Snooper/src/main/java/com/prateekj/snooper/utils/Logger.java
package com.prateekj.snooper.utils;

import android.util.Log;

public class Logger {
  private static final String SNOOPER_DEBUGGER_TAG = "AndroidSnooper";

  public static void d(String tag, String message) {
    Log.d(SNOOPER_DEBUGGER_TAG + tag, message);
  }

  public static void e(String tag, String message, Throwable exception) {
    Log.e(SNOOPER_DEBUGGER_TAG + tag, message, exception);
  }

  public static void e(String tag, String message) {
    Log.e(SNOOPER_DEBUGGER_TAG + tag, message);
  }
}

File: Snooper/src/main/java/com/prateekj/snooper/utils/Logger.kt
package com.prateekj.snooper.utils

import android.util.Log

object Logger {
  private const val SNOOPER_DEBUGGER_TAG = "AndroidSnooper"

  fun d(tag: String, message: String?) {
    Log.d(SNOOPER_DEBUGGER_TAG + tag, message)
  }

  fun e(tag: String, message: String?, exception: Throwable) {
    Log.e(SNOOPER_DEBUGGER_TAG + tag, message, exception)
  }

  fun e(tag: String, message: String?) {
    Log.e(SNOOPER_DEBUGGER_TAG + tag, message)
  }
}
